Minecraft.Client/Blocks/Lava.cs

```diff
@@ -4,4 +4,14 @@ public class Lava : Block
     {
         public Lava() : base(BlockType.Lava) { }
     }
+
```

Comment:
So I think the semantics are still that "flowing lava is lava" so there should be inheritance if only as a semantic thing. What I don't quite understand is (because I haven't checked yet) is what the behavior of BlockType.Lava is: is it flowing or stationary, and do we need three different classes, and if we don't, how to keep 1:1 fidelity with what's going on in Minecraft.